man/systemd.resource-control.xml

```diff
@@ -1023,6 +1023,34 @@ RestrictNetworkInterfaces=~eth1</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BindNetworkInterface=</varname></term>
+
+        <listitem>
+          <para>Takes the name of a network interface. This option causes every socket created by processes of this
+          unit to be bound to the specified network interface.
+          </para>
+
+          <para>It is specially useful to confine a process to a VRF, when the program does not offer native support
+          for it. This was previously achieved by prepending the value in the <varname>ExecStart=</varname> line with
```

Comment:
I'd really rework the previously achieved thing, I'd just say that this is equivalent to "ip vrf exec".